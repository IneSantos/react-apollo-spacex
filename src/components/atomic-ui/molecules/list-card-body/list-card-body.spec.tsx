import React from "react";
import { render } from "@testing-library/react";
import ListCardBody from "./list-card-body";

describe("ListCardBody molecule tests", () => {
  test(`ListCardBody component render`, () => {
    const { container } = render(<ListCardBody details="This is a mission detail" date="2022/04/10"/>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
