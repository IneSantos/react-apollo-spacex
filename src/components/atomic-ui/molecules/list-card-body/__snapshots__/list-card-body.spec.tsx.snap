// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListCardBody molecule tests ListCardBody component render 1`] = `
<div
  class="list-card-body"
>
  <div
    class="details"
  >
    This is a mission detail
  </div>
  <div
    class="launch-date"
  >
    April 10, 2022
  </div>
</div>
`;
